drivers/docker/config.go

```diff
@@ -491,6 +510,16 @@ type DockerVolumeDriverConfig struct {
 	Options hclutils.MapStrStr `codec:"options"`
 }
 
+type ContainerGCConfig struct {
```

Comment:
It may be useful to have two modes of operation for this, detect+log vs detect+shutdown - this would let operators safely enable it to build trust before letting a reaper loose on their production infrastructure.

Reply:
Excellent suggestion - I'll do that.  Thanks!